README.md

```diff
@@ -196,25 +245,29 @@ root of this repo.
 
 To run it against a deployed cluster:
 
-1. `bundle install`
-
-2. Set YUL_DC_SERVER to the domain name for your deployed cluster
+1. Set YUL_DC_SERVER to the domain name for your deployed cluster
 `export YUL_DC_SERVER=collections-test.curationexperts.com`
 
-3. Ensure http basic auth credentials for Blacklight are set in your
-`.secrets` file (See [secrets-template](./secrets-template) for an
-example). Otherwise the test suite will set its user and password
-to what is in env vars before defaulting to 'test'. To set via env
-vars:
+2. Set the HTTP loging vars:
    - Set HTTP_USERNAME to the known Blacklight http basic auth
    username for your deployed cluster `export
-   HTTP_USERNAME=<basic-auth-username>` - Set HTTP_PASSWORD to the
+   HTTP_USERNAME=<basic-auth-username>` 
+   - Set HTTP_PASSWORD to the
    known Blacklight http basic auth password for you deployed cluster
    `export HTTP_PASSWORD=<basic-auth-password>`
-4. `rspec spec/deploy_spec.rb`
 
-## Releasing a new Camerata version
+3. `cam smoke`
+
+## Releasing a new dependency version
+
+1. Follow the release process laid out in the application README through to completion
+2. Set the version variable to the new version with `cam release APP_NAME VERSION_NUMBER`
+3. Start the applications with the new version and run the smoke test
+4. Deploy the applications (see deployment above) Example:
+`cam deploy-main.sh yul-test`
```

Comment:
take off `.sh` here?

Reply:
yep, missed that one. it was in my clipboard when I fixed the others.  One thing I did for the transition is that camerata allows the .sh versions to make it easier for folks who already remember them.